src/main.rs

```diff
@@ -311,12 +311,32 @@ fn run_test(cx: &Context, args: &Args) -> Result<()> {
         cargo.arg("-Z");
         cargo.arg("doctest-in-workspace");
     }
-    cargo::test_args(cx, args, &mut cargo);
 
-    if term::verbose() {
-        status!("Running", "{}", cargo);
+    if args.ignore_run_fail {
+        let mut cargo_no_run = cargo.clone();
+        cargo_no_run.arg("--no-run");
+        cargo::test_args(cx, args, &mut cargo_no_run);
+        if term::verbose() {
+            status!("Running", "{}", cargo_no_run);
+        }
+        cargo_no_run.stdout_to_stderr().run()?;
+
+        cargo.arg("--no-fail-fast");
+        cargo::test_args(cx, args, &mut cargo_no_run);
+        if term::verbose() {
+            status!("Running", "{}", cargo);
+        }
+        if let Err(e) = cargo.stdout_to_stderr().run() {
+            warn!("{}", e);
+        }
```

Comment:
Hmm, If a warning exists, it is a bit ugly because the same warning is duplicated in the first run (no-run) and the second run (no-fail-fast).

Reply:
Changed to hide output from the first run (no-run), if the verbose output is not enabled.